### ISSUE TITLE
hallucination?

### ISSUE THREAD

**Issue by erleyva** [WGSExtract/WGSExtract.github.io#24]
I've been looking into this for about 2 hours now and I'm reaching a knowledge plateau. However, what I see seems to point towards something not working right and bringing a false positive.

While checking out Promethease, I was getting a positive 35x for Crohn's Disease. rs2066847(C;C)
![image](https://github.com/WGSExtract/WGSExtract.github.io/assets/152233013/1d5eeece-97af-4be1-ac62-b6ba1e5e8709)

I wasn't able to find that snp on gene.iobio.io which is the platform that nebula uses to explore mutations. So I checked the Mirror file produced by WGS Extract, and I see the SNP twice. Not only that, but it also has the old SNP rs5743293 twice which was merged into rs2066847 according to https://www.snpedia.com/index.php/rs5743293.
![image](https://github.com/WGSExtract/WGSExtract.github.io/assets/152233013/ab3a61ae-adce-47da-84ad-4da38dd5d0fa)

I found it odd that the SNP shows up twice with 2 different codes, CC (Crohn's Disease) and GG (No problemo). So I'm still wondering, am I in trouble or not lol.

Then I went to explore the genome itself on IGV using the CRAM file to get as accurate as I could. went to the same position where this mutation is supposed to be found at position Chr16, 50729867. 
 https://www.snpedia.com/index.php/Rs2066847. I see an interesting situation there. it's mostly GG, but there is a C and a T. I double checked on the Nebula genome browser and sure enough, same thing.

![image](https://github.com/WGSExtract/WGSExtract.github.io/assets/152233013/eb8741cd-6705-4a16-bbe6-ec8da8d4b7a9)

I'm thinking the expected result in this case would be to show only the GG variant. I'm also fairly new at this, so there could be something I didn't properly understand.

**Comment by erleyva** [comment]
Thank you so much for looking into it. I appreciate the explanation on why duplicates happen and the different versions.

Yes, I meant Microarray when I wrote mirror.

I'm a little confused by the build 37 and build 38 piece. My CRAM references Build 38. There was a note in the manual saying this shouldn't be a problem.

Please correct me if I'm wrong here. But what I'm gathering is that the combined kit should still be the best microarray file to upload into Promethease

**Comment by RandyHarr** [comment]
WGSE will call using the build38 reference for a build38 BAM/CRAM.  And then do a liftover of the values back to build37. So the coordinate positions given should be for 37. Your image:
![image](https://github.com/WGSExtract/WGSExtract.github.io/assets/58348151/cf3c8a29-8d74-44bf-8375-b836e33740e4)
indicated build38 coordinate, microarray format file.  Was not sure how you got a hold of that.
CombinedKit is better for any site that can accept more than a basic microarray file result.  It helps provide a greater coverage of the genome and alleles that a tool may look for.  But, as you discovered, it could introduce confusion if the different lab vendor versions have conflicting information. We should likely do a better job to weed those out as CombinedKit is an artificial microarray file not coming from any particular lab.

**Comment by erleyva** [comment]
I see what you mean now. I think I see where those came from. The first time I ran the extract, something went wrong and either I had to force close WGSE, or it wasn't responding anymore, can't remember which one. However, the combined kit file did get created. That's the one I posted the screenshot of (inadvertedly, as I meant to screenshot the second file when I generated again which didn't have any issues). I'd speculate that the conversion to build37 weren't done yet when I force closed the process. Now that I opened the "good file", I see different positions which are probably the build 37 ones.

![image](https://github.com/WGSExtract/WGSExtract.github.io/assets/152233013/c5457bed-b500-4c73-a1bd-6df29688f105)
